components/admin_console/license_settings/enterprise_edition/enterprise_edition_right_panel.test.tsx

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {shallow} from 'enzyme';
+import React from 'react';
+
+import {LicenseSkus} from 'mattermost-redux/types/general';
+
+import EnterpriseEditionRightPanel, {EnterpriseEditionProps} from './enterprise_edition_right_panel';
+
+describe('components/admin_console/license_settings/enterprise_edition/enterprise_edition_right_panel', () => {
+    const license = {
+        IsLicensed: 'true',
+        IssuedAt: '1517714643650',
+        StartsAt: '1517714643650',
+        ExpiresAt: '1620335443650',
+        SkuShortName: LicenseSkus.Starter,
+        Name: 'LicenseName',
+        Company: 'Mattermost Inc.',
+        Users: '1000000',
+        IsGovSku: 'false',
+    };
+
+    const props = {
+        isTrialLicense: false,
+        license,
+    } as EnterpriseEditionProps;
+
+    test('should render for no Gov no Trial no Enterprise', () => {
```

Comment:
Could these be changed to assert on specific parts of the render output? That would make easier to debug if it breaks in the future, and less likely to blindly do a `npm run test --update`.

It might require using `mountWithIntl` (e.g.  `import {mountWithIntl} from 'tests/helpers/intl-test-helper';`) instead of `shallow` if what's being asserted on is deep in the component hierarchy, and/or if it uses internationalized strings.

Reply:
Agreed! Just updated them.